docs/reference-sequence-import.md

```diff
@@ -0,0 +1,119 @@
+# Importing Reference Sequence
+
+This document shows the flow of how different types of ReferenceSequences are extracted and imported.
+
+## Types of Reference Sequence to import
+   
+* GENOME_ASSEMBLY
+* SEQUENCE
+* TRANSCRIPTOME_SHOTGUN_ASSEMBLY
+ 
+## Retrieving Reference Sequence Accessions
+
+SRA Analysis records can contain a reference sequence in either of the three analysis categories:
+* REFERENCE_ALIGNMENT
+* SEQUENCE_VARIATION 
+* PROCESSED_READS
+
+It is guaranteed that each analysis belong to at most one of these three types.
+Example XML below which is parsed into a Java object:
+ 
+```xml
+<SEQUENCE_VARIATION>
+     <ASSEMBLY>
+         <STANDARD accession="GCA_000001405.1"/>
+     </ASSEMBLY>
+     <SEQUENCE accession="CM000665.1" label="chr3"/>
+     [...]
+</SEQUENCE_VARIATION>
+```
+
+### Retrieving ASSEMBLY accession
+The Assembly type reference sequence accession is represented in the XML as shown below:
```

Comment:
Minor: `Assembly` is referred in smalls inside and `SEQUENCE`, `TRANSCRIPTOME_SHOTGUN_ASSEMBLY` in caps. Can be made consistent.